units/host_allocator.go

```diff
@@ -246,6 +247,33 @@ func (j *hostAllocatorJob) Run(ctx context.Context) {
 	hostQueueRatio := float32(timeToEmpty.Nanoseconds()) / float32(distroQueueInfo.MaxDurationThreshold.Nanoseconds())
 	noSpawnsRatio := float32(timeToEmptyNoSpawns.Nanoseconds()) / float32(distroQueueInfo.MaxDurationThreshold.Nanoseconds())
 
+	lowRatioThresh := float32(.25)
```

Comment:
Can this be a constant?

Reply:
Also just wondering, but in the ticket you mentioned .5; why did you change it?

Reply:
I looked at a bunch of scenarios where linux distros were entering terminal decline, and decided that a more conservative rule was more appropriate. I originally wanted to have some sort of rule like "has been falling for 15 minutes consecutively" but the more conservative threshold also means that we don't need to worry about keeping state